src/ServiceControl/Operations/FailedAuditImportCustomCheck.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ServiceControl.Operations
+{
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using Raven.Client;
+    using Raven.Client.Indexes;
+    using System;
+    using System.Linq;
+
+    class FailedAuditImportCustomCheck : CustomCheck
+    {
+        public FailedAuditImportCustomCheck(IDocumentStore store)
+            : base("Audit Message Ingestion", "ServiceControl Health", TimeSpan.FromHours(1))
+        {
+            this.store = store;
+        }
+
+        public override CheckResult PerformCheck()
+        {
+            using (var session = store.OpenSession())
+            {
+                var query = session.Query<FailedAuditImport, FailedAuditImportIndex>();
+                using (var ie = session.Advanced.Stream(query))
+                {
+                    if (ie.MoveNext())
+                    {
+                        var message = @"One or more audit messages have previously failed to import properly into ServiceControl and have been stored in the ServiceControl database.
```

Comment:
I think this message needs to be slightly reworded but let's wait until we have the re-processing ready to see what should it be.

Reply:
Reworded